client/src/main/java/io/pravega/client/netty/impl/FlowHandler.java

```diff
@@ -237,6 +269,8 @@ public void close() {
                 if (openFlowCount != 0) {
                     log.warn("{} flows are not closed", openFlowCount);
                 }
+                log.debug("clearing Append Batch size Trackers {}", idBatchSizeTrackerMap);
```

Comment:
"Clearing".

Also have you checked what the output of this log instruction is? You are calling `toString` on a `Map`. That will likely just tell you how many elements are in the map.

Reply:
this change is removed now.. along with flow , the append size tracker map entries are removed.